optimum/onnxruntime/utils.py

```diff
@@ -234,6 +236,30 @@ def check_io_binding(providers: List[str], use_io_binding: Optional[bool] = None
     return use_io_binding
 
 
+def get_ordered_input_names(input_names: Dict[str, int], func: Callable) -> List[str]:
```

Comment:
```suggestion
def get_ordered_input_names(input_names: List[str], func: Callable) -> List[str]:
```
Maybe we should do this because there is no need for the values here.

Reply:
thanks!